app/views/assessments/statistics.html.erb

```diff
@@ -1,10 +1,9 @@
 <% @title = "Statistics" %>
 <h2 id="statsHeading">Statistics for <%= link_to @assessment.display_name, course_assessment_path(@course, @assessment) %></h2>
 <% if @assessment.problems.count > 0 %>
-<%= javascript_include_tag "https://code.jquery.com/jquery-2.1.4.min.js" -%>
 
-<%= javascript_include_tag "https://cdn.plot.ly/plotly-latest.min.js" -%>
```

Comment:
Dead code since plotting code was removed in https://github.com/autolab/Autolab/commit/20274fd60497db917394b986571da2fea90c919b (supposedly meant to be readded in v2, but it never was)

In fact, all the `<div id='xxxDivGraph'>` appear to be dead code.

In fact, it seems